crates/nu-cmd-lang/src/core_commands/describe.rs

```diff
@@ -76,6 +76,7 @@ impl Command for Describe {
                 example: "'hello' | describe",
                 result: Some(Value::test_string("string")),
             },
+            /*
```

Comment:
Curious why are these examples commented out?

Reply:
Because nu_cmd_lang can not depend on nu_command...
And these 2 particular tests are referencing a command "Each" which is located in nu_command.
After this stuff lands we can think about moving "describe" up to nu_command.
Or we can possibly move Each down into nu_cmd_lang

Reply:
Believe it or not out of all of the tests this is the only place where there was an issue like this 😄

Reply:
Ok, thanks for the explanation.